kvirt/kvm/__init__.py

```diff
@@ -456,7 +456,13 @@ def create(self, name, virttype=None, profile='kvirt', flavor=None, plan='kvirt'
                 ipv6.append(netname)
             else:
                 return {'result': 'failure', 'reason': "Invalid network %s" % netname}
-            ovsxml = "<virtualport type='openvswitch'/>" if ovs else ''
+            if ovs:
```

Comment:
ovsxml needs to be defined and set to blank when ovs is set to False

Reply:
ACK, pardon.